src/Wallabag/ImportBundle/Import/InstapaperImport.php

```diff
@@ -115,6 +117,22 @@ public function parseEntry(array $importedEntry)
         $entry->setUrl($importedEntry['url']);
         $entry->setTitle($importedEntry['title']);
 
+        if($importedEntry['tags']) {
+            foreach(explode(',', $importedEntry['tags']) as $tag) {
+                if(strlen($tag) === 0)continue;
+                $existingTag = $this->em
+                    ->getRepository('WallabagCoreBundle:Tag')
+                    ->findOneByLabel($tag);
```

Comment:
There is a performance issue with this method, see https://github.com/wallabag/wallabag/issues/2864. 
We need to work on it.